work-in-progress/chc-comp/batch-convert-to-sygus.sh

```diff
@@ -0,0 +1,82 @@
+#!/usr/bin/env bash
+
+set -Euo pipefail
+
+if (( ${BASH_VERSION%%.*} < 4 )); then echo "ERROR: [bash] version 4.0+ required!" ; exit -1 ; fi
+
+USER_DIR="$(pwd)"
+ROOT_DIR="$(cd -P -- "$(dirname -- "${BASH_SOURCE[0]}")" && pwd -P)/.."
+
+SYGUS_EXTENSION="sl"
+SMT_EXTENSION="smt2"
+SOURCE_SYGUS_STANDARD="1"
+TARGET_SYGUS_STANDARD="2"
+
+IS_VERBOSE=""
+DO_COLLECT_ERRORS=""
+OUTPUT_DIR=""
+INTERNAL_OPTIONS=""
+
+usage() {
+  if [ -n "$1" ]; then echo -e "\nERROR: $1" >&2 ; fi
+  echo -en "
+Usage: $0 [flags] -o <dir> <dir>
+
+Parameters:
+    [--output-dir, -o]          The output directory for converted files
+
+Flags:
+    [--verbose, -v]             Increase output verbosity (list PASSsed files too)
+" 1>&2 ; exit 1
+}
+
+for opt in "$@"; do
+  shift
+  case "$opt" in
+    "--output-dir")         set -- "$@" "-o" ;;
+    "--verbose")            set -- "$@" "-v" ;;
+    "--")             set -- "$@" "--" ;;
```

Comment:
May be leave an empty line above this one so the main flags stay separated above (and the indentation doesn't look strange).